src/pango.rs

```diff
@@ -16,7 +17,8 @@ pub use self::widgets::{
     Item,
     Rectangle,
     Matrix,
-    GlyphString
+    GlyphString,
+    FontDescription
```

Comment:
Rust style recommends trailing commas in such cases to reduce churn in the future (note how the above line had to change because it lacked the comma).